data/categories.txt

```diff
@@ -646,16 +646,16 @@ el:Αρτοποιείο|κατάστημα
 sk:3Pekáreň|obchod
 sw:Kituo cha kuwoka mikate
 
-shop
+shop|@shop
```

Comment:
shop is an osm key
@shop contains translations for keys from the shop category

"shop" belongs to the "@shop" category